alpheios_nemo_ui/data/templates/alpheios/mainnavmenu.html

```diff
@@ -1,42 +1,49 @@
-                    <span class="m-dropdown-item-text">{{crumb.title}}</span>
-                    {% if crumb.current %}
-                       <span class="m-dropdown-item-text-current">({{ crumb.current }})</span>
-                    {% endif %}
-                </a>
-            {% endif %}
-          {% endfor %}
-        </div>
-        <div class="m-dropdown-content">
-          <a class="dropdown-item m-dropdown-item" href="https://alpheios.net">
-            <span class="m-dropdown-item-icon">{% include "alpheios::icons/External-Link-Icon.svg" %}</span>
-            <span class="m-dropdown-item-text">Go to Alpheios.net</span>
-          </a>
-        </div>
-        <div class="main-dropdown-login">
-            {% include "alpheios::logout.html" %}
-        </div>
-        
-        </div>
\ No newline at end of file
+    <div class="main-dropdown-logo">
+        <a href="{{ url_for('.r_index') }}"><img class="logo" src="{{url_for('.secondary_assets', filetype='static', asset='alpheios-logo.svg')}} " alt="Alpheios Reading Center" /></a>
+        <h1>Reading Center</h1>
+    </div>
+
+    <div class="main-dropdown-header">
+      <a href="javascript:void(0)" class="toolbar-button-icon" data-dismiss="modal" id="dropdownMenuButtonClose">{% include "alpheios::icons/X-Close-Icon.svg" %}</a>
+      <h3 class="main-dropdown-title">Main Menu</h3>
+    </div>
+
+    <div class="main-dropdown-login">
+        {% include "alpheios::login.html" %}
+    </div>
+
+    <div class="m-dropdown-content">
+      <p class="main-dropdown-content-title">Collections</p>
+      {% for crumb in menu: -%}
+        {% if loop.index < 4 %}
+          {% set args = { 'objectId': crumb['args']['objectId'] } %}          
+            <a class="dropdown-item m-dropdown-item" href="{{url_for(crumb.link, **args)}}">
+                <span class="m-dropdown-item-icon" >{% include "alpheios::icons/Collections-Folder-Icon.svg" %}</span>
+                <span class="m-dropdown-item-text">{{crumb.title}}</span>
+                {% if crumb.current %}
+                    <span class="m-dropdown-item-text-current">({{ crumb.current }})</span>
+                {% endif %}
+            </a>
+        {% endif %}
+      {% endfor %}
+    </div>
+    <div class="m-dropdown-content">
+      <a class="dropdown-item m-dropdown-item" href="https://alpheios.net">
+        <span class="m-dropdown-item-icon">{% include "alpheios::icons/External-Link-Icon.svg" %}</span>
+        <span class="m-dropdown-item-text">Go to Alpheios.net</span>
+      </a>
+    </div>
+    <div class="main-dropdown-logout">
+        {% include "alpheios::logout.html" %}
+    </div>
+
+    <div class="main-dropdown-logo-bottom">
+        <a href="{{ url_for('.r_index') }}">
+          <img class="logo" src="{{url_for('.secondary_assets', filetype='static', asset='alpheios-logo.svg')}} " alt="Alpheios Reading Center" />
+          <h4>Reading Center</h4>
```

Comment:
what if we just put the icon next to the alpheios.net link above?

Reply:
It would be like this
![image](https://user-images.githubusercontent.com/12377640/57113647-d3ecc480-6d88-11e9-9049-27fdf979488c.png)